libs/blocks/modal/modal.js

```diff
@@ -20,18 +21,15 @@ export function findDetails(hash, el) {
 }
 
 function fireAnalyticsEvent(event) {
-  // eslint-disable-next-line no-underscore-dangle
-  window._satellite?.track('event', {
+  const data = {
     xdm: {},
-    data: {
-      web: { webInteraction: { name: event?.type } },
-      _adobe_corpnew: { digitalData: event?.data },
-    },
-  });
+    data: { web: { webInteraction: { name: event?.type } } },
+  };
+  if (event?.data) data.data._adobe_corpnew.digitalData = event.data;
```

Comment:
Wouldn't this throw an error? On L26 you define `data.data`, which in turn doesn't have the `_adobe_corpnew` property, so we'd get a `Cannot set properties of undefined (setting 'digitalData')` error. Am I missing something?

Reply:
@overmyheadandbody That's a good point. Updated.